opm/parser/eclipse/EclipseState/Schedule/ScheduleEnums.hpp

```diff
@@ -119,6 +119,7 @@ namespace Opm {
     namespace WellProducer {
 
         enum ControlModeEnum {
+            NONE =     0,
```

Comment:
Maybe a comment here for when `NONE` applies - I guess that is only for `WHISTCTL`.